include/swoole.h

```diff
@@ -411,7 +411,31 @@ typedef struct
     socklen_t len;
 } swSocketAddress;
 
-typedef struct _swConnection
+typedef struct _swConnection swConnection;
+typedef struct _swEvent swEvent;
+typedef struct _swReactor swReactor;
+typedef struct _swProtocol swProtocol;
+
+struct _swProtocol
```

Comment:
no changed so much. Only move `struct _swProtocol` ahead of `struct _swConnection`